packages/core/interfaces/index.ts

```diff
@@ -106,6 +106,14 @@ export interface UseContent<CONTENT, SEARCH> {
   error: any
 }
 
+export interface UseLocale<SET_LANGUAGE, GET_LANGUAGE, SET_COUNTRY, GET_COUNTRY, GET_COUNTRIES> {
+  setLanguage: SET_LANGUAGE,
+  getLanguage: GET_LANGUAGE,
+  setCountry: SET_COUNTRY,
+  getCountry: GET_COUNTRY,
+  getCountries: GET_COUNTRIES
```

Comment:
`setLanguage`, `setCountry`, `country`, `availableCountries`, `language`, `availableLanguages`

Reply:
or - even better, use just `country` and `country` and use reactivity - when I assign a new lang (`language.value = 'en') it will be automatically populated